packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -34,7 +34,7 @@
     "@types/mocha": "^5",
     "@types/node": "12.0.12",
     "@types/sinon": "^2.3.7",
-    "@types/vscode": "1.49.0",
+    "@types/vscode": "1.72.0",
```

Comment:
Note I bumped the version of vscode to 1.72. This is the version released in 10/2021. Verified via appsights that our user base is keeping vscode up to date.  This was b/c I wanted to make use of the contextExtension.extension.packageJSON value for getting the contents of the package.json file instead of reading it directly.